tools/cibadmin.c

```diff
@@ -726,16 +723,7 @@ main(int argc, char **argv)
         goto done;
     }
 
-    if (options.cib_action == cib_action_EMPTYCIB  /* pointer comparison */) {
-        output = createEmptyCib(1);
-        if (optind < argc) {
-            crm_xml_add(output, XML_ATTR_VALIDATION, argv[optind]);
-        }
-        pcmk__xml_serialize_fd_formatted(STDOUT_FILENO, output);
```

Comment:
You still want to replace the dump_xml_formatted() call, it's just in the strcmp(options.cib_action, "empty") block in the current code.